### ISSUE TITLE
take screenshot on key press on emulator

### ISSUE THREAD

**Issue by aliafshany** [sakebook/pbssc#2]
hello there
thank you for this amazing repo.
could you also do something when we press a key like Volume Up buton, the script automatically take a SC?

best

**Comment by sakebook** [comment]
@aliafshany 

Hello. Thank you for using.

> could you also do something when we press a key like Volume Up buton, the script automatically take a SC?

No. You need to run script from terminal.

**Comment by aliafshany** [comment]
Yes, I tested that, however as you know, I'm taking screenshots from an epub reader app, and I need to take screenshots from each page. On emulator, you may turn the page by pressing the Up Volume key.

Could you please implement such a feature?

**Comment by sakebook** [comment]
Hmm, I have no plans to implement that feature.

If you want to take to screenshot by pressing the Up Volume key, you run pbscc from some script every second.
